code/wdb/writedown.q

```diff
@@ -0,0 +1,22 @@
+/-endofperiod function to savedown by integer
+endofperiod:{[currp;nextp;data]
+	.lg.o[`endofperiod;"Received endofperiod. currentperiod, nextperiod and data are ",(string currp),", ", (string nextp),", ", .Q.s1 data]
+        /-check if there is more than 1 wdb processes
+	h:exec w from .servers.SERVERS where proctype=`wdb,not w=0N;
+        /-create a dictionary of process names and their start times
+	times:raze{@[;".proc.starttimeUTC";()][x]} each h;
```

Comment:
`times:raze@[;".proc.starttimeUTC";()]each h;`
We don't need the function brackets here